packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -389,4 +389,14 @@ module.exports = {
       }
     }
   },
-}
+  async ready() {
```

Comment:
This is a really good find -- I didn't know we had the ability to iterate on `ctx.pages` and then update the content per-page like this. This may be helpful in the future for page-specific header content.